### ISSUE TITLE
Neo v3.2.1 Checklist

### ISSUE THREAD

**Issue by superboyiii** [neo-project/neo#2676]
Now We're making the checklist of neo v3.2.1 for release preparation. We plan to release v3.2.1 and then upgrade testnet and mainnet. It's an enhancement version including native contract upgrade, these core issues and PRs are included. 

**Mandatory issues**
- [x] Testnet can break after ContractManagement's update https://github.com/neo-project/neo/issues/2673
- [x] all RPC nodes with TokensTracker module on testnet stopped syncing https://github.com/neo-project/neo/issues/2670
- [x] all RPC nodes on testnet stopped syncing maybe caused by ApplicationLogs https://github.com/neo-project/neo/issues/2666
- [x] Running nccs at master gives Unhandled Exception https://github.com/neo-project/neo-devpack-dotnet/issues/721

**Mandatory PRs**

~~Versioned Abi https://github.com/neo-project/neo/pull/2632~~ (need better solution)
- [x] Fix paths https://github.com/neo-project/neo-devpack-dotnet/pull/724
- [x] Improve CompilationContext.ContractName https://github.com/neo-project/neo-devpack-dotnet/pull/723
- [x] Add ToJson overload https://github.com/neo-project/neo/pull/2671
- [x] Fix oom https://github.com/neo-project/neo/pull/2665
- [x] Limit result stack https://github.com/neo-project/neo-modules/pull/696
- [x] limit free gas https://github.com/neo-project/neo-modules/pull/697
- [x] Add new testnet network id https://github.com/neo-project/neo-node/pull/852
- [x] Optimize inventory https://github.com/neo-project/neo/pull/2659
- [x] add murmur32 to crypto lib https://github.com/neo-project/neo/pull/2604
- [x] Fix fee in NativeContract.Invoke https://github.com/neo-project/neo/pull/2692
- [x]  GetTransaction signers https://github.com/neo-project/neo/pull/2685
- [x] Allow the committee to block a member from the committee https://github.com/neo-project/neo/pull/2695
- [x] fix MaxBlockSystemFee https://github.com/neo-project/neo-modules/pull/703/ 
- [x]  Add oracle global timeout https://github.com/neo-project/neo-modules/pull/698

@neo-project/core @neo-project/developers @neo-project/ngd-seattle

**Comment by roman-khimov** [comment]
Is #2628 also a part of the release?

**Comment by superboyiii** [comment]
> Is #2628 also a part of the release?

Yes, these already merged will be included in the release.

**Comment by devhawk** [comment]
@superboyiii What's the timeframe on releasing 3.2? I have been letting https://github.com/neo-project/neo-modules/pull/687 languish, but can get it ready by middle of next week for inclusion in 3.2. Would enable a set of new features we want to bring to Neo.Express. But I don't think we should hold up the release for https://github.com/neo-project/neo-modules/pull/687.

**Comment by superboyiii** [comment]
> @superboyiii What's the timeframe on releasing 3.2? I have been letting [neo-project/neo-modules#687](https://github.com/neo-project/neo-modules/pull/687) languish, but can get it ready by middle of next week for inclusion in 3.2. Would enable a set of new features we want to bring to Neo.Express. But I don't think we should hold up the release for [neo-project/neo-modules#687](https://github.com/neo-project/neo-modules/pull/687).

Enough time left. We wish in 2 weeks.

**Comment by devhawk** [comment]
I realize it is coming in late, but I would really like #2679 to be considered for 3.2.

Also, https://github.com/neo-project/neo-modules/pull/687 is just waiting on review. Can we add that to the 3.2 list?

**Comment by devhawk** [comment]
I hadn't noticed before that neo-modules repo has `master` and `develop` branches. Are we going to merge `develop` into `master` for 3.2 release? Or is `develop` the new default branch for neo-modules repo?

**Comment by superboyiii** [comment]
> I hadn't noticed before that neo-modules repo has `master` and `develop` branches. Are we going to merge `develop` into `master` for 3.2 release? Or is `develop` the new default branch for neo-modules repo?
> 
> https://github.com/neo-project/neo-modules/branches

`develop` will be merged into master before release.

**Comment by roman-khimov** [comment]
neo-project/neo-modules#698? Or is it considered as a part of #2666?

**Comment by superboyiii** [comment]
> [neo-project/neo-modules#698](https://github.com/neo-project/neo-modules/pull/698)? Or is it considered as a part of #2666?

No time left for this, can be included in next releae.

**Comment by superboyiii** [comment]
> [neo-project/neo-modules#698](https://github.com/neo-project/neo-modules/pull/698)? Or is it considered as a part of #2666?

We will wait for it to be merged into v3.2.1,

**Comment by devhawk** [comment]
When is https://github.com/neo-project/neo-devpack-dotnet going to be updated to neo 3.2.1?

**Comment by superboyiii** [comment]
> When is https://github.com/neo-project/neo-devpack-dotnet going to be updated to neo 3.2.1?

Wait for https://github.com/neo-project/neo-devpack-dotnet/pull/731
